build/rbac/gen-common.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+set -eEuo pipefail
+set -x
```

Comment:
why is this separated from previous `set`?

Reply:
🤷‍♀️